build.sbt

```diff
@@ -75,3 +70,15 @@ lazy val checkerPlugin = (projectMatrix in file("checker-plugin"))
   .settings(libraryDependencies ++= Seq(akkaTyped % Test, akkaPersistence % Test, akkaProjections % Test))
   .dependsOn(checkerLibrary)
   .jvmPlatform(scalaVersions = supportedScalaVersions)
+
+lazy val schemaDumpPlugin = (projectMatrix in file("sbt-dumpschema"))
+  .enablePlugins(SbtPlugin)
+  .settings(name := "sbt-dumpschema")
+  .settings(commonSettings)
+  .settings(pluginCrossBuild / sbtVersion :=
+    virtualAxes.value
+      .collectFirst { case x: ScalaVersionAxis => x.value }
+      .map { case "2.12" => "1.2.8" }
+      .getOrElse("1.5.2"))
+  .dependsOn(checkerLibrary)
```

Comment:
Question: Is depending on the same annotation as checker-plugin a good idea? There are other ways of achieving the same functionality(like using sbt setting keys) but I wonder which is the best for the end-user.

Reply:
Hmm let's stick to the single way of doing stuff as for now indeed 🤔